README.md

```diff
@@ -319,7 +319,7 @@ Boolean - default 0
 
 added to provide minimal automated checking of responses
 
-$spotify->die\_on\_response\_error(1);
+    $spotify->die_on_response_error(1);
```

Comment:
The `README.md` is auto-generated by `dzil` on release, so you don't need to update it manually. That saves a bit of time.